lib/Settings/Admin.php

```diff
@@ -21,25 +21,31 @@
 
 namespace OCA\LogReader\Settings;
 
+use OCA\LogReader\Service\SettingsService;
 use OCP\AppFramework\Http\TemplateResponse;
-use OCP\IL10N;
+use OCP\AppFramework\Services\IInitialState;
 use OCP\Settings\IDelegatedSettings;
+use OCP\Util;
 
 class Admin implements IDelegatedSettings {
-	/** @var IL10N */
-	private $l;
+	private IInitialState $initialState;
+	private SettingsService $settingsService;
```

Comment:
```suggestion
```

Reply:
Not sure why I GitHub can't separate multi-line suggestions for deleted lines and new lines but basically the suggestions are addressing the same block.

https://github.com/nextcloud/logreader/blob/118731f3c39d592e4c9e7e013278620fce7d20bb/lib/Settings/Admin.php#L31C1-L37C3


which could simply be replaced by :

```php
public function __construct(private IInitialState $initialState, private SettingsService $settingsService) {
	}
```